### PR TITLE
<Carousel /> - add unidriver

### PR DESCRIPTION
**Pull request by justenor**

### CODE REVIEW

File: src/Carousel/Carousel.uni.driver.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const withoutClonedNodes = (selector = '') =>
+  `.slick-slide:not(.slick-cloned) ${selector}`;
+
+export const carouselUniDriverFactory = base => {
+  return {
+    ...baseUniDriverFactory(base),
+    isLoading: async () => (await base.$$('[data-hook="loader"]').count()) > 0,
+    getChildren: () => base.$$(withoutClonedNodes()).map(child => child),
+    getImages: () =>
+      base.$$(withoutClonedNodes('[data-hook="carousel-img"]')).map(img => img),
```
Comment (mosheKerbel) [inline src/Carousel/Carousel.uni.driver.js]: looks like the old driver is returning the src?
https://github.com/wix/wix-style-react/blob/a34bcbe8ab2611d3194669e02ac045f76bf0b912/src/Carousel/Carousel.driver.js#L19

**Comment by mosheKerbel** [comment]
u got a conflict in the spec

File: src/Carousel/Carousel.private.uni.driver.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        .$$('[data-hook="carousel-img"]')
+        .map(async img => ReactTestUtils.Simulate.load(await img.getNative()));
+    },
+    clickPrevious: () => base.$('[data-hook="prev-button"]').click(),
+    clickNext: () => base.$('[data-hook="next-button"]').click(),
+    mouseOver: () => {
+      const imageContainer = base.$('.slick-current');
+      ReactBase(imageContainer).mouseEnter();
+    },
+    mouseOut: () => getCurrentSlide().click(),
```
Comment (mosheKerbel) [inline src/Carousel/Carousel.private.uni.driver.js]: name implie it shoud be mouseOut and not click

File: src/Carousel/Carousel.private.uni.driver.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    loadImages: () => {
+      base
+        .$$('[data-hook="carousel-img"]')
+        .map(async img => ReactTestUtils.Simulate.load(await img.getNative()));
+    },
+    clickPrevious: () => base.$('[data-hook="prev-button"]').click(),
+    clickNext: () => base.$('[data-hook="next-button"]').click(),
+    mouseOver: () => {
+      const imageContainer = base.$('.slick-current');
+      ReactBase(imageContainer).mouseEnter();
```
Comment (mosheKerbel) [inline src/Carousel/Carousel.private.uni.driver.js]: name implies is should be mouseOver and not mouseEnter

File: src/Carousel/Carousel.uni.driver.js (hunk 1 above)
Comment (justenor) [reply src/Carousel/Carousel.uni.driver.js]: fixed

File: src/Carousel/Carousel.private.uni.driver.js (hunk 2 above)
Comment (justenor) [reply src/Carousel/Carousel.private.uni.driver.js]: Fixed

File: src/Carousel/Carousel.private.uni.driver.js (hunk 3 above)
Comment (justenor) [reply src/Carousel/Carousel.private.uni.driver.js]: Fixed

**Pull request merged**